### PR TITLE
[zh] resync pages in reference

### PR DESCRIPTION
**Pull request by Zhuzhenghao**
This PR resynced some pages in `content/zh-cn/docs/reference`, including:

1. Update the page weight
2. Clean up the markdown style

### CODE REVIEW

File: content/zh-cn/docs/reference/glossary/service.md [hunk 1]
```diff
@@ -28,18 +28,32 @@ tags:
...
 服务确保可以将网络流量定向到该工作负载的当前 Pod 集合。
+
+<!--
+Kubernetes Services either use IP networking (IPv4, IPv6, or both), or reference an external name in
+the Domain Name System (DNS).
+
+The Service abstraction enables other mechanisms, such as Ingress and Gateway.
+-->
+
+Kubernetes Service 要么使用 IP 网络（IPv4、IPv6 或两者），要么在域名系统 (DNS) 中引用外部名称。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/glossary/service.md:57]:
```suggestion
Kubernetes Service 要么使用 IP 网络（IPv4、IPv6 或两者），要么引用位于域名系统 (DNS) 中的外部名称。
```

**Comment by tengqm** [comment]
The build is not successful. Please try rebase to `main` and push again.

File: content/zh-cn/docs/reference/access-authn-authz/admission-controllers.md [hunk 2]
```diff
@@ -1123,36 +1123,26 @@ PodNodeSelector 允许 Pod 强制在特定标签的节点上运行。
...
-[Pod 安全标准](/zh-cn/docs/concepts/security/pod-security-standards/)来确定是否可以执行请求。
-
-更多信息请参阅 [Pod 安全性准入控制器](/zh-cn/docs/concepts/security/pod-security-admission/)。
-
-### PodSecurityPolicy {#podsecuritypolicy}
-
-{{< feature-state for_k8s_version="v1.21" state="deprecated" >}}
+PodSecurity 准入控制器在新 Pod 被准入之前对其进行检查，
+根据请求的安全上下文和 Pod 所在命名空间允许的
+[Pod 安全性标准](/docs/concepts/security/pod-security-standards/)的限制来确定新 Pod
```
Comment (ydFu) [inline content/zh-cn/docs/reference/access-authn-authz/admission-controllers.md:1133]:
```suggestion
[Pod 安全性标准](/zh-cn/docs/concepts/security/pod-security-standards/)的限制来确定新 Pod
```

**Review by ydFu — COMMENTED** [review]
Localized content must use localized links.

**Review by ydFu — APPROVED** [review]
/lgtm

**Comment by windsonsea** [comment]
Thanks.
/lgtm

**Pull request merged**